run_create_s3_srr.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+import json
+from argparse import ArgumentParser
+
+from run_common import AWSCli
+from run_common import _confirm_phase
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('-i', '--origin_bucket_account_id', type=str, required=True, help='origin bucket account id')
+    parser.add_argument('-o', '--origin_bucket_name', type=str, required=True, help='origin bucket name')
+    parser.add_argument('-r', '--replication_bucket_name', type=str, required=True, help='replication bucket name')
+    parser.add_argument('-a', '--replication_aws_access_key', type=str, required=True,
+                        help='Replication bucket AWS ACCESS KEY ID')
+    parser.add_argument('-s', '--replication_aws_secret_key', type=str, required=True,
+                        help='Replication bucket AWS SECRET ACCESS KEY')
+    parser.add_argument('-p', '--srr_policy_name', type=str, required=True, help='write down the policy name you want')
+    parser.add_argument('-n', '--srr_role_name', type=str, required=True, help='write down the role name you want')
+
+    args = parser.parse_args()
+
+    _confirm_phase()
+
+    return args
+
+
+def create_replication_bucket(args):
```

Comment:
@GoldieKMG // 인자를 args 로 쓰는 것보다 `replication_bucket_name, origin_bucket_account_id` 로 하는게 더 명확 할 것 같 습니다. 그리고 함수 명도 기능에 맞게 작성하면 좋을 것 같습니다.
```suggestion
def put_policy_replication_bucket(replication_bucket_name, origin_bucket_account_id):
```